katello_certs_tools/fileutils.py

```diff
@@ -284,10 +283,10 @@ def createPath(path, user='apache', group='apache', chmod=0755):
             sys.stderr.write("Changing owner for %s failed\n" % path)
 
 
-def setPermsPath(path, user='apache', group='root', chmod=0750):
+def setPermsPath(path, user='apache', group='root', chmod='0750'):
```

Comment:
`os.chmod` (as used in line 308) does not accept string parameters (at least not in Python 3.6 and 3.7)